packages/collector/src/announceCycle/unannounced.js

```diff
@@ -240,6 +247,28 @@ function applyIgnoreEndpointsConfiguration(agentResponse) {
   agentOpts.config.tracing.ignoreEndpoints = configNormalizers.ignoreEndpoints.normalizeConfig(ignoreEndpointsConfig);
 }
 
+/**
+ * Apply global stack trace configuration from the agent response.
+ *
+ * @param {AgentAnnounceResponse} agentResponse
+ */
+function applyStackTraceConfiguration(agentResponse) {
+  const globalConfig = agentResponse?.tracing?.global;
+  if (!globalConfig) return;
+
+  ensureNestedObjectExists(agentOpts.config, ['tracing', 'global']);
```

Comment:
// TODO: Later we can add local handling here (if local is present -> we take this, otherwise we take global)